### ISSUE TITLE
In interactive debug mode, FPC contents cannot be displayed

### ISSUE THREAD

**Issue by abekornelis** [z390development/z390#515]
Please add a command to display the FPC, analogous to PSW and PSW+

**Comment by abekornelis** [comment]
John Ganci suggested the following (original email edited for brevity):

It would also be good to add FPC to DUMP and SNAP output. 

Attached is a text file showing 
- proposed new test command "fpc" and its output
- proposed DUMP and SNAP output for the floating-point registers.

My plan would be to add a new sz390.java method dump_fpc(boolean flag). Put it above or below the dump_fpr() method.The argument would determine whether to produce the output for the test command (" FPC=xxxxxxxx") or output for DUMP/SNAP ("  FPC  xxxxxxxx"; lines up with following FPR lines). Sample output shown below.

Suggested new z390 test command and its output:
```
fpc
test cmd: fpc
 FPC=00000000
```

Suggested z390 new DUMP
```
EZ390E error  11 ABEND PSW=07850600 800FFED8 0A0D         SVC   ABEND U0100
 R0-R3 00000000800FFEA0 0000000000000064 0000000000002304 0000000000000000
 R4-R7 0000000000000064 0000000000000000 0000000000000000 0000000000000000
 R8-RB 0000000000000000 0000000000000000 0000000000000000 0000000000000000
 RC-RF 00000000800FFEA0 0000000000002100 0000000000002018 00000000800FFEA0

  FPC  00000000                                                             <--- new
 F0-F3 0000000000000000 0000000000000000 0000000000000000 0000000000000000
 F4-F7 0000000000000000 0000000000000000 0000000000000000 0000000000000000
 F8-FB 0000000000000000 0000000000000000 0000000000000000 0000000000000000
 FC-FF 0000000000000000 0000000000000000 0000000000000000 0000000000000000
```

Suggested z390 new SNAP output:
```
SNAP DUMP ID=1 TEXT=PDATA=(FPR)
  FPC  00000000                                                            <--- new
 F0-F3 0000000000000000 0000000000000000 0000000000000000 0000000000000000
 F4-F7 0000000000000000 0000000000000000 0000000000000000 0000000000000000
 F8-FB 0000000000000000 0000000000000000 0000000000000000 0000000000000000
 FC-FF 0000000000000000 0000000000000000 0000000000000000 0000000000000000
SNAP DUMP ID=2 TEXT=PDATA=(REGS)
 R0-R3 0000000060000002 00000000000FFF7C 0000000000000000 0000000000000000
 R4-R7 0000000000000000 0000000000000000 0000000000000000 0000000000000000
 R8-RB 0000000000000000 0000000000000000 0000000000000000 0000000000000000
 RC-RF 00000000800FFF00 00000000000FFFB8 00000000000FFFB8 00000000800FFF00
  FPC  00000000                                                            <--- new
 F0-F3 0000000000000000 0000000000000000 0000000000000000 0000000000000000
 F4-F7 0000000000000000 0000000000000000 0000000000000000 0000000000000000
 F8-FB 0000000000000000 0000000000000000 0000000000000000 0000000000000000
 FC-FF 0000000000000000 0000000000000000 0000000000000000 0000000000000000
```

**Comment by jyganci** [comment]
Fixed in PR #517.
